test/theta_sketch_dup_test.cc

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <gtest/gtest.h>
+#include <iomanip>
+#include <fstream>
+#include "theta_sketch_dup.hpp"
+
+namespace datasketches {
+
+TEST(THETA_SKETCH_DUP, TEST) {
```

Comment:
Could you update all the tests with EXPECT, ASSERT ect and remove cout and commented content? 
Reference http://cheezyworld.com/wp-content/uploads/2010/12/PlainGoogleQuickTestReferenceGuide1.pdf